Multisig/UI/UI Library/TokenImage.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  TokenImage.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 14.05.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+import struct Kingfisher.KFImage
+
+struct TokenImage: View {
+    var imageURL: URL
+    var size: CGFloat = 28
+
+    var body: some View {
+        KFImage(imageURL)
+            .placeholder {
+                Image(systemName: "arrow.2.circlepath.circle")
```

Comment:
It doesn't look pretty 😄 I think it is better to use the icon from the previous project.

Reply:
The final design will come with the next pr. This is a placeholder.